apps/docusaurus/docs/guides/keyless-accounts.md

```diff
@@ -1,53 +1,127 @@
+3. Set up the appropriate callback URL (https://your-app-domain.com/auth/google/callback) for handling authentication responses from Google.
+
+### Registering your dApp with Apple
+
+**Step 1: Sign in to Apple Developer Account**
+
+1. Go to the [Apple Developer website](https://developer.apple.com/).
+2. Sign in with your Apple ID.
+
+**Step 2: Create a New App ID**
+
+1. Navigate to the "Certificates, Identifiers & Profiles" section.
+2. Click on "Identifiers" in the sidebar.
+3. Click the "+" button to create a new App ID.
+4. Fill in the details for your app, including the name and bundle ID.
+5. Enable "Sign in with Apple" under the "Capabilities" section.
+6. Click "Continue" and then "Register" to create the App ID.
+
+**Step 3: Generate a Private Key**
+
+1. In the "Keys" section of the "Certificates, Identifiers & Profiles" page, click the "+" button to create a new key.
+2. Enter a name for the key, enable the "Sign in with Apple" capability, and click "Continue."
+3. Download the generated private key and securely store it. This key will be used to authenticate your app with Apple's OIDC service.
+
+**Step 4: Configure Redirect URIs**
+
+1. Under the "App ID" section, locate your newly created App ID and click on it.
+2. Scroll down to the "Sign in with Apple" section and click on "Edit."
+3. Add the redirect URIs that your application will use for callback after authentication. The format should be https://your-app-domain.com/auth/apple/callback.
+4. Click "Save" to update the settings.
+
+**Step 5: Set Up Your OIDC Integration**
+
+1. Use an OIDC library or framework compatible with Apple's OIDC service (e.g., Passport.js for Node.js, Spring Security for Java).
+2. Configure your application to use the client ID and private key obtained from Apple during the registration process.
+3. Set up the appropriate callback URL (https://your-app-domain.com/auth/apple/callback) for handling authentication responses from Apple.
+
+## Aptos Keyless Integration Guide
 
 :::info Only devnet and testnet is supported
-Currently Aptos Keyless is only supported in devnet and testnet. Mainnet support to come in the following weeks.
+Currently Aptos Keyless is only supported in devnet and testnet. Mainnet support scheduled for June 7.
+:::
+
+:::tip Keyless Account Scoping
+Use of the **_Aptos Keyless Integration Guide_** will allow for the integration of keyless accounts directly into your application. This means that blockchain accounts are scoped to your application's domain (logging in with your Google account on dApp A and logging in with your Google account on dApp B will create separate accounts). Stay tuned for more to come on Aptos’ plan to allow Keyless accounts to be used portably across applications.
+
+To provide feedback, get support, or be a design partner as we enhance Aptos Keyless, join us here: https://t.me/+h5CN-W35yUFiYzkx
 :::
```

Comment:
You need to use the Callout component. `:::` is only for docusaurus. See https://preview.aptos.dev/en/developer-platforms/contribute/components/callout